src/overlays/actors/ovl_En_Paper/z_en_paper.c

```diff
@@ -184,14 +230,16 @@ void EnPaper_Draw(Actor* thisx, PlayState* play) {
             gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 64, 255, 128, 255);
         }
 
-        if ((this->actor.floorHeight - 40.0f) < ptr->unk_18.y) {
-            MtxF* mf = GRAPH_ALLOC(play->state.gfxCtx, sizeof(MtxF));
+        if ((this->actor.floorHeight - 40.0f) < piece->pos.y) {
+            Mtx* mtx = GRAPH_ALLOC(play->state.gfxCtx, sizeof(Mtx));
 
-            func_8017842C(mf, this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, ptr->unk_30, ptr->unk_0C.x,
-                          ptr->unk_0C.y, ptr->unk_0C.z, ptr->unk_18.x, ptr->unk_18.y, ptr->unk_18.z);
+            //! @bug This function does not necessarily form a correct rotation, see its documentation for details.
```

Comment:
Personally I don't think the bug tag is warranted. It is enough to have the bug tag on the actual function itself.

Reply:
Probably fair.